components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -46,6 +49,26 @@ export default class ConfirmIntegration extends React.Component {
         }
     }
 
+    copyLink = (e) => {
```

Comment:
These methods should also call `e.preventDefault()` to prevent a '#' from being added to the URL